flight/Modules/VibrationAnalysis/vibrationanalysis.c

```diff
@@ -116,7 +118,8 @@ static void VibrationAnalysisCleanup(void) {
         PIOS_Thread_Delete(taskHandle);
         taskHandle = NULL;
     }
-    
+
+#ifdef PIOS_FREE_IMPLEMENTED
```

Comment:
This isn't strictly necessary, but it's still okay, you can call free even though it does nothing. ;-)